routing/dht/routing.go

```diff
@@ -28,6 +29,8 @@ var asyncQueryBuffer = 10
 // PutValue adds value corresponding to given Key.
 // This is the top level "Store" operation of the DHT
 func (dht *IpfsDHT) PutValue(ctx context.Context, key key.Key, value []byte) error {
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10) // CR: should this be removed instead?
```

Comment:
> Does this mean all dht ops or some dht ops (just PutValue, GetValue, etc)?

if we're going to time out the dht ops, we should timeout all of them. before adding new timeouts,  let's maybe consider it in a different PR, so have more time to think about it. (for now just up this timeout to 30s)